docs/about/whats-new.ipynb

```diff
@@ -748,6 +799,21 @@
     "A unit for the linear part is currently not supported."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### `broadcast` supports `DataArray`\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**New in 0.13**\n",
+    "\n",
+    "`broadcast` now also supports data arrays.\n",
+    "\n",
+    "</div>"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {
```

Comment:
Add something about being able to do operations between `DataArray` and `float`/`int`?

Reply:
ok fine